tagger/interrogator.py

```diff
@@ -212,54 +218,57 @@ def interrogate(
         return ratings, tags
 
 
+def get_onnxrt():
+    try:
+        import onnxruntime
+        return onnxruntime
+    except ImportError:
+        # only one of these packages should be installed at a time in any one environment
+        # https://onnxruntime.ai/docs/get-started/with-python.html#install-onnx-runtime
+        # TODO: remove old package when the environment changes?
```

Comment:
In what scenario do you expect the environment to change? installed on an external drive?
I don't think we should support this.